correcting grammatical errors on transcript 31 (#142)
mgreiler/se-unlocked: Transcripts/EP_38_Michael_Feather.md

from all over the world. This is so, so fascinating. I'm super curious
about how different organizations develop software. I'm always asking the
questions.
What makes teams and organization more effective than others? What's up
engineering practices. Do we have, I'm a big fan of code reviews,
 from your experience, are there best practices? Can you make out best
practices that really lead to success for engineering teams where you say, well,
if they follow that, right, they  will be very successful and also factors that
you think are definitely bad and lead to a lot of troubles.

**Michael Feathers:**[00:01:23] The question. And it's interesting too, because
like the practice space is very interesting, but I think a lot of how it really
comes down to organizational culture, you know? And it's, you know, if you have
a good culture, then basically like the practices will develop almost
invariably, right. Or you'll at least be open to going in and exploring
different practices and things along those lines.
I think,for the thing that I have gotten called into organizations
for quite often,legacy code, the thing I kind of noticed over and

> over again is that. What is missing sometimes is a frank

Reviewer: ```suggestion
over again is that. What is missing sometimes is really a very frank
```

Right. And in the worst cases, it's kind of like a, It's like everybody is told,
you know, you must write code and you must design this thing and create it, but
nobody's really paying attention to it. And sort of like, you know, thinking
about it as being significant, quite often there's like a task focus
rather than focusing on the quality of the thing that you're producing.
So I wouldn't really know how to go and actually sort of say, what
are the best practices? I think that so many of the things that we basically do.
In the industry now regarding testing and pairing and mobbing and you know, the
way that we allocate work and stuff like that, a lot of those things really kind
of help.

**Michaela:**: [00:02:38] Yeah. So one thing that I thought about is engineering
values versus practices, right? So I think that engineering values and
developing dos is a team. And not a lot of people are talking about it.
Unfortunately I think much more people should talk about values. The engineering
values that we have and not the practices because practices can and
should change.
Right. They should change over time. And with technology changes and

> how our society changes, the practices should change new ideas

Reviewer: ```suggestion
how our society changes, the practices should change. And somebody comes out and has a new idea, right, and they 
```

> introduced.They try something, they fail maybe 10 times, and

Reviewer: ```suggestion
try something. They fail maybe 10 times, and
```
